instant/flags.vim

```diff
@@ -124,3 +124,17 @@ call s:plugin.Flag('uncovered_guibg', 'red')
 ""
 " Text color for the uncovered lines when in GUI mode (e.g. gvim).
 call s:plugin.Flag('uncovered_guifg', 'white')
+
+""
+" For the gcov plugin, directories under which to shallowly search for gcov data
+" files.
+call s:plugin.Flag('gcov_search_paths', ['.'])
```

Comment:
These flags are unused now and can be removed for now, right?

Reply:
d'oh, yup, removed